frontend/src/utils/metadata.ts

```diff
@@ -0,0 +1,107 @@
+import type { AudioDetail, ImageDetail, Metadata } from "~/types/media"
+import { IMAGE } from "~/constants/media"
+
+import type { NuxtI18nInstance } from "@nuxtjs/i18n"
+
+const getImageType = (
+  imageType: string | undefined,
+  i18n: NuxtI18nInstance
+) => {
+  if (imageType) {
+    if (imageType.split("/").length > 1) {
+      return imageType.split("/")[1]
+    }
+    return imageType
+  }
+  return i18n.t("mediaDetails.information.unknown")
+}
+
+const getAudioType = (audio: AudioDetail, i18n: NuxtI18nInstance) => {
+  if (!audio.alt_files)
+    return audio.filetype ?? i18n.t("mediaDetails.information.unknown")
+  const altFormats = audio.alt_files.map((altFile) => altFile.filetype)
+  if (audio.filetype) {
+    altFormats.unshift(audio.filetype)
+  }
+  const uniqueFormats = new Set(altFormats)
+  return [...uniqueFormats].join(", ")
+}
+
+export const getMediaMetadata = (
```

Comment:
If this function is branched over image and audio it might be clearer if it called one of two separate functions depending on the media type.

Reply:
I've just converted what we were doing in the `VAudioDetails` and `VImageDetails` components to a util function here. It's improved (and synced between image and audio metadata) in the follow-up PR.